### PR TITLE
Updates css to cover edge cases for the engineering sprint conference

### PR DESCRIPTION
**Pull request by petesfrench**
## Done

- Adds style to cover the elements that were not styled for in the original Ubuntu Summit project.
These are:
Date inputs (registration form)
image
Registration count (homepage)
image
Participation list table (participation list)
image

- Fixes some minor visual bugs

## QA

- Setup and run the indico project as described in [the readme](https://github.com/canonical/canonical-indico-themes)
- If you have access to canonicals inidco account, you can log in and render the events as they are on live. If you do not, you will have to create an event and add the associated elements, for example: to get a date input on the registration form, create a registration form and add a input for date. If you have any trouble creating the appropriate elements, please reach out.
- Make sure the highlighted issues have been dealt with and flag any other issue you may find (this plugin was styled specifically for the Ubuntu Summits needs and does not cover every possible usage of the app)

## Issue

Fixes https://warthogs.atlassian.net/browse/WD-2595

### CODE REVIEW

File: indico_themes_canonical/static/css/ubuntu-summit/registration.css [hunk 1]
```diff
@@ -97,7 +97,35 @@ i.icon.icon-phone {
...
+}
+
+button.SingleDatePickerInput_calendarIcon {
+  padding: 0;
+}
+
+/* Yes/No */
+
+.events-registration-client-styles-___regform-module__boolean-field___0geJs .ui.button {
+  background: var(--brand-off-white);
```
Comment (kewisch) [inline indico_themes_canonical/static/css/ubuntu-summit/registration.css:112]: Can we set both a `background` and a `color` here? This makes it more explicit and avoids adverse effects where the `color` might be white as well in a different context.

File: indico_themes_canonical/static/css/ubuntu-summit/registration.css [hunk 2]
```diff
@@ -97,7 +97,35 @@ i.icon.icon-phone {
...
+  font-size: 1rem;
+  font-weight: 400;
+}
+
+.events-registration-client-styles-___regform-module__boolean-field___0geJs .ui.button.active {
+  background: var(--brand-warm-grey);
```
Comment (kewisch) [inline indico_themes_canonical/static/css/ubuntu-summit/registration.css:118]: Same here

**Review by kewisch — APPROVED** [review]
r+ with these comments addressed:

File: indico_themes_canonical/static/css/ubuntu-summit/registration.css (hunk 1 above)
Comment (petesfrench) [reply indico_themes_canonical/static/css/ubuntu-summit/registration.css:112]: Good catch, just updated it

**Comment by petesfrench** [comment]
@kewisch Could you take another look at this please

**Comment by petesfrench** [comment]
@carkod Would you be able to take a look at this please (I think you have Indico set up)? Philip is away until next week

**Pull request merged**